src/circuits/runcircuit.jl

```diff
@@ -163,9 +163,11 @@ Apply a quantum gate to an MPS/MPO.
 function applygate!(M::Union{MPS,MPO},
                     gatename::String,
                     sites::Union{Int,Tuple};
+                    cutoff=1E-15,
+                    maxdim=1_000,
```

Comment:
Again, not a fan of putting an explicit maximum dimension. Also a reminder that we want to remove `applygate` in favor of `runcircuit`.

Reply:
I still need to fix a few things around, including this.

Reply:
I will also add the `runcircuit` with a single gate